docs/guides/architecture-scenarios.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Architecture Scenarios
+description: This guide outlines a number of the common architecture scenarios for building applications with Clerk, their characteristics, and limitations.
+---
+
+# Architecture Scenarios
+
+There are several ways to model how users and organizations fit into your application. The 3 scenarios that will be covered in this guide are:
+
+1. B2C: Business to Consumer
+2. B2B: Business to Business
+3. Platforms
+
+We will share some of the common characteristics of apps in each scenario as well as the level of support for these features in Clerk.
+
+## B2C: Business to Consumer
+
+B2C companies focus on selling products or services directly to consumers. Some popular examples are Netflix, Headspace, and Spotify.
+
+The B2C user management model is the simplest of the three and it is the default model supported by Clerk out of the box, with little-to-no configuration.
+
+In a B2C scenario, applications generally share the following characteristics:
+
+- A user creates a single account with your service
+- There is a single, shared user-pool which all the users belong to
+- Any connections enabled for your application are available to all users to authenticate with
+- The application branding is that of your company (i.e.: not white-labelled per customer or organization)
+- The application is accessible under a single domain (e.g.: `example.com` or `app.example.com`)
+
+**TODO: diagram of B2C app with Clerk**
+
+<Callout type="info">
+In the B2C scenario, organizations are generally not necessary since users that sign-up to your application typically do not exist as part of a team, organization, or workspace.
```

Comment:
In this case, "sign up" is a verb and not a noun, so the hyphen is not needed!

```suggestion
In the B2C scenario, organizations are generally not necessary since users that sign up to your application typically do not exist as part of a team, organization, or workspace.
```